api/models.py

```diff
@@ -127,7 +127,7 @@ def skills(self):
     condition_immunities = models.TextField()
     senses = models.TextField()
     languages = models.TextField()
-    challenge_rating = models.TextField(help_text='Monster challenge rating.')
+    challenge_rating = models.TextField()
```

Comment:
It's taking this help text out intentional?

Reply:
Yes, this caused a migration to be created, then a migration conflict. That conflict was resolved elsewhere, but the model didn't get dleted.  That line is totally rewritten elsewhere anyway, so basically, it's a very temporary reversion of something that shouldn't have been in this PR anyway.